### PR TITLE
Update domain.pddl

### PR DESCRIPTION
**Pull request by jzylf**
67行与原71行重复

### CODE REVIEW

**Review by pluehne — CHANGES_REQUESTED** [review]
@jzylf: Thank you so much for your contribution! Well spotted, this line is indeed redundant.

My unwritten policy is that the original domains and instances should not be modified, because I consider them the ground truth, and I don’t want them to be semantically altered. (I want the domains and instances in this repository to exactly represent what was run at the original competitions.)

With that said, I think that your modification is well justified. After all, the problem’s semantics aren’t changed at all.

What I’d suggest, though, is to add a note saying

>Duplicate `(power_on ?i)` entries in conjunctive action preconditions were removed.

to the [domain’s readme](https://github.com/potassco/pddl-instances/tree/master/ipc-2002/domains/satellite-strips-hand-coded).

Aside from that, note that this duplicate entry is also present in the following domain variants:

- [Satellite (STRIPS, For Automatic Planners)](https://github.com/potassco/pddl-instances/blob/master/ipc-2002/domains/satellite-strips-automatic/domain.pddl)
- [Satellite (Numeric, For Automatic Planners)](https://github.com/potassco/pddl-instances/blob/master/ipc-2002/domains/satellite-numeric-automatic/domain.pddl)
- [Satellite (Numeric, For Hand-Coded Planners)](https://github.com/potassco/pddl-instances/blob/master/ipc-2002/domains/satellite-numeric-hand-coded/domain.pddl)
- [Satellite (Numeric (Hard), For Automatic Planners)](https://github.com/potassco/pddl-instances/blob/master/ipc-2002/domains/satellite-numeric-hard-automatic/domain.pddl)

To sum it up, I’d like to request the following changes:

- [ ] repeat your change for the other domain variants listed above
- [ ] add the note that I suggested above to all those domain’s readme files identically
- [ ] please give your commits a meaningful English commit message

I’d be glad if you would address my suggestions :smiley:.
